django_irods/views.py

```diff
@@ -157,10 +152,7 @@ def download(request, path, rest_call=False, use_async=True, use_reverse_proxy=T
             if is_zip_request:
                 daily_date = datetime.datetime.today().strftime('%Y-%m-%d')
                 output_path = "zips/{}/{}/{}.zip".format(daily_date, uuid4().hex, path)
-                if res.is_federated:
-                    irods_output_path = os.path.join(res.resource_federation_path, output_path)
-                else:
-                    irods_output_path = output_path
+                irods_output_path = os.path.join(res.resource_federation_path, output_path)
```

Comment:
`irods_output_path = res.irods_path`

Reply:
@sblack-usu ```irods_path``` is a property on ```ResourceFile``` class, not on ```BaseResource``` class, right? So we cannot really use it on the resource object.

Reply:
you're right.

Reply:
Can this be updated in here too?  blocks like this:
https://github.com/hydroshare/hydroshare/blob/develop/hs_core/views/resource_folder_hierarchy.py#L122